src/panels/config/integrations/ha-integration-card.ts

```diff
@@ -360,6 +380,13 @@ export class HaIntegrationCard extends LitElement {
           align-items: center;
           padding-right: 5px;
         }
+        .card-actions .documentation {
+          color: var(--primary-text-color);
+        }
+        .card-actions .documentation ha-svg-icon {
+          color: var(--secondary-text-color);
+          margin-left: 5px;
+        }
```

Comment:
You can remove this, `meta` of `mwc-list-item` will take care of it.

Reply:
Done. Also added the following to styles as recommended below:
```
 
ha-button-menu {
          --mdc-menu-min-width: 200px;
        }
```